### DISCUSSION TITLE
Feature -  Windows GUI

### DISCUSSION THREAD

**Discussion by blublub** [simulot/immich-go#310] [General]
Hello
I know this is probably not a very popular request/opinion but in order for a wide and easy adoption of immich a simple GUI version of immich go would be very useful as Windows still is the most used OS and the vast majority of ppl dont know how to us the cli.
I know that anyone who can install immich in the current state can do it, but further down the road things will hopefully smooth out for a broader adoption and immich-go is extremely useful and a great tool to get started with already stored images.

A simple approach would be sufficient:
enter hostname
api key
browser path
tick option the most standard / usefull options (folder/album creation, dry-run etc..)
open log button
same visual presentation during operation

**Comment by simulot** [comment]
Yes, it is popular...

I'm not a professional programmer. This is a huge step for me. 
Then it should be done for Mac OS too...
Help wanted!

**Reply by blublub** [reply]
Sadly I cannot code :-(
